lib/commands/web.js

```diff
@@ -269,12 +273,59 @@ extensions.translateWebCoords = async function (coords) {
 };
 
 extensions.checkForAlert = async function () {
+  try {
+    const alertText = await this.proxyCommand('/alert/text', 'GET');
+    if (util.hasValue(alertText)) {
+      log.debug(`Found an alert with text '${alertText}'`);
+      return true;
+    }
+  } catch (ign) {}
   return false;
 };
 
 extensions.waitForAtom = async function (promise) {
-  // TODO: Add check for alert and accept/dismiss it as per autoAcceptAlert capability
-  return this.parseExecuteResponse(await promise);
+  this.activeAlertHandlers = this.activeAlertHandlers || [];
```

Comment:
What is the case that there would be more than 1 active alert handler?

Reply:
the checked is going to be unique per driver instance. That is why we have an array here. Each try to start a new checker simply pushes a new array item and stop checker pops one item. We only start new checker instance if this array is empty and stop it after the last element has been removed from it.

Reply:
I have renamed the variable to avoid confusion

Reply:
I guess I still don't understand the need for this. At any given time for a particular driver instance there can only be _one_ command being executed. So there will never be more than one handler. Or am I missing something?

Reply:
this is kind'a safety net for me. If you say it is an overkill and such situation is never supposed to happen then I could change it back to a simple boolean flag

Reply:
Can the alert handler not return a cancelable promise, and use that instead? Otherwise a boolean would make more sense to me.